src/gtpV2Codec/gtpV2Stack.cpp

```diff
@@ -45,6 +45,48 @@ GtpV2Stack::~GtpV2Stack ()
     // TODO Auto-generated destructor stub
 }
 
+/******************************************
+ * Add Seq as key and ue_idx as value to map
+ ******************************************/
```

Comment:
GtpV2Stack is an auto generated file. So next time we run the script these changes will be lost. We should move this to a different file